week2/index.js

```diff
@@ -4,14 +4,56 @@ var options = process.argv.slice(2);
 
 var command = options[0];
 
+
 switch (command) {
   case 'help':
-  default:
-    showHelp();
+    help();
     break;
   case 'list':
     listTodos();
     break;
+  case 'add':
+    add()
+    break;
+  case 'remove':
+    remove()
+    break;
+  case 'reset':
+    reset()
+    break;
+  case 'update':
+    update()
+    break;
+  default:
+    showHelp();
```

Comment:
Why not just `help()` as above?
Even better - how can we change this case to execute for both `help` and the default?
(DRY - don't repeat yourself)